bindings/java/src/main/native/context.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+#ifndef CONTEXT_H_
+#define CONTEXT_H_
+
+#include <ucp/api/ucp.h>
+
+#include <atomic>
+
+
+/**
+ * Context wrapper - allocates and releases ucp context
+ */
+class context {
+public:
+    context() : ucp_context(nullptr), ref_count(0) {}
+
```

Comment:
why 2 space lines? can use just one (in multiple places)

Reply:
Second-to-last bullet in [coding style](https://github.com/openucx/ucx/blob/master/doc/CodeStyle.md#style). Should I change also proto-types to 1 space line?

Reply:
maybe it was confusing, byt in case of C++, the whole class is considered a "type" so this bullet does not refer to the member functions declaration inside the class. anyway they should be sep by 1 line.